### PR TITLE
[NEW] REST function to fetch list of members of a group

### LINKED ISSUE #64: Adding rest function to get list of group members
To add function to get list of all the members in a group. It will be similar to get all the members in a channel which is already made.

### PR DESCRIPTION
**Pull request by srrathi**
members list is used to show members while adding @ mentioning
Resolve #64

### CODE REVIEW

File: rest/group_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		assert.GreaterOrEqual(t, len(members), 1)
+
+	})
+
+	t.Run("With channel ID", func(t *testing.T) {
+		general := &models.Channel{ID: "GENERAL"}
+		members, err := rocket.GetGroupMembers(general)
+		assert.Nil(t, err)
+		assert.GreaterOrEqual(t, len(members), 1)
+	})
```
Comment (debdutdeb) [inline rest/group_test.go:26]: `general` is not a group but channel. The difference is subtle, both are considered "rooms" but public ones are channels and private ones are groups. In this particular case `groups.*` endpoints do not check any public rooms, i.e. channels, which general is.

**Review by debdutdeb — CHANGES_REQUESTED** [review]
The channel/group distinction needs to be fixed on the SDk at a deeper level, and is out of this pr's scope/goals. For now, adding a test for this will need creating a private group first - then deleting it, ofc.

File: rest/group_test.go (hunk 1 above)
Comment (debdutdeb) [reply rest/group_test.go:26]: Just wanted to give some visual example
```sh
$ culr https://open.rocket.chat/api/v1/groups.members\?roomId\=GENERAL
{"success":false,"error":"The required \"roomId\" or \"roomName\" param provided does not match any group [error-room-not-found]","errorType":"error-room-not-found"}%
$ culr https://open.rocket.chat/api/v1/channels.members\?roomId\=GENERAL -s | jq .success
true
```

Comment (srrathi) [reply rest/group_test.go:26]: you are right. General is a channel and using it to get members of a group will eventually fail. To add test for it I can make a private group run tests on it and then delete it. As I see function to create private group exists in realtime API so I think I can use it.

**Review by debdutdeb — APPROVED** [review]

**Pull request merged**